test/module/libs/common/combine_latest_until_first_completed_test.cpp

```diff
@@ -5,9 +5,9 @@
 
 #include "common/combine_latest_until_first_completed.hpp"
 
-#include <gtest/gtest.h>
 #include <chrono>
 
+#include <gtest/gtest.h>
```

Comment:
No, I **suggested** place gtest **after** boost. Otherwise, it is not clang-formatted.

>
>
> <img width="479" alt="image" src="https://user-images.githubusercontent.com/36847043/53229027-1e203e00-3694-11e9-868a-03687acef0be.png">

Reply:
It is the way `clang-format 7.0.1-6` places them with our config.